net/pfSense-pkg-cellular/files/usr/local/bin/cellular_dev.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/local/bin/python2.7
+#
+# /usr/local/bin/cellular_dev.py
+#
+# part of pfSense (https://www.pfsense.org)
+# Copyright (C) 2017 Voleatech GmbH
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+Helper to mount cellular modem at fixed dev point
+"""
+
+import sys
+import os
+import commands
+
+import argparse
+
+data_file = "cuaZ99.0"
+data_path = "/dev/" + data_file
+mgm_file = "cuaZ99.1"
+mgmt_path = "/dev/" + mgm_file
+
+def remove_link():
+
+    if os.path.lexists(data_path):
+        os.system("rm " + data_path)
```

Comment:
Why do this instead of using os.remove() / os.unlink()? Using native functions is better than shell exec. Same applies to other instances using this syntax.